wai-aria/states-and-properties/aria-hidden.html

```diff
@@ -0,0 +1,170 @@
+<!doctype html>
+<html>
+  <head>
+    <title>aria-hidden Tests</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+    <style>
+      .sr-only {
+        clip: rect(1px, 1px, 1px, 1px);
+        clip-path: inset(50%);
+        height: 1px;
+        width: 1px;
+        margin: -1px;
+        overflow: hidden;
+        padding: 0;
+        position: absolute;
+      }
+    </style>
+  </head>
+  <body>
+    <main id="main-element">
+    <h1><code>aria-hidden</code> Tests</h1>
+    <p>Tests the <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/2090.html#aria-hidden">aria-hidden</a>
+      ARIA state and its potential values (e.g., <code>true</code>, <code>false</code>, <code>undefined</code>)</p>
+    <!-- Possible aria-hidden values:
+      - "true" = The element is hidden from the accessibility API
+      - "false" = The element's hidden state is determined by the user agent based on whether it is rendered. Synonym of undefined
+      - "undefined" = synonym of "false"
+
+    General web techniques related to showing/hiding content:
+      Rendered:
+        - display: <anything besides 'none'> = rendered
+        - visibility: visible = rendered
+        - clipped/off-screen text = rendered
+        - inert = rendered (but hidden from assistive technologies)
+        - aria-hidden = rendered (but hidden from assistive technologies)
```

Comment:
should this include 'true' so it's not misread that the attribute, regardless of state, hides content?

```suggestion
        - aria-hidden=true = rendered (but hidden from assistive technologies)
```